<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Двухпросмотровый ассемблер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainLayout">
    <item row="2" column="2">
     <widget class="QLabel" name="setupLabel">
      <property name="text">
       <string>Таблица настройки</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="secondSetupTable"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="secondPassErrorsLabel">
      <property name="text">
       <string>Ошибки второго прохода</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="secondPassLabel">
      <property name="text">
       <string>Двоичный код</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="firstPassErrorsLabel">
      <property name="text">
       <string>Ошибки первого прохода</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="firstPassTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTextEdit" name="secondPassErrorsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="firstPassButton">
        <property name="text">
         <string>Первый проход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="secondPassButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Второй проход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="choiseExampleLabel">
        <property name="text">
         <string>Выбор примера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="exampleComboBox">
        <item>
         <property name="text">
          <string>Прямая адресация</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Относительная адресация</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Смешанная адресация</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="secondPassTextEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="tsiTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="firstPassLabel">
      <property name="text">
       <string>Вспомогательная таблица</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="tsiLabel">
      <property name="text">
       <string>Таблица символических имен</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="firstPassErrorsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="6">
     <widget class="QGroupBox" name="leftGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	border: 0px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="sourceCodeLabel">
         <property name="text">
          <string>Исходный текст</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="sourceCodeTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="commandsLabel">
         <property name="text">
          <string>Таблица кодов операций</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="commandsTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
